bridge/package.json

```diff
@@ -52,10 +53,12 @@
     "truffle-plugin-verify": "^0.5.9"
   },
   "devDependencies": {
+    "@nomiclabs/hardhat-etherscan": "^2.1.6",
     "axios": ">=0.21.1",
     "coveralls": "^3.1.1",
     "css-what": ">=5.0.1",
     "elliptic": ">=6.5.4",
+    "ethereum-waffle": "^3.4.0",
```

Comment:
what is this dependency for?

Reply:
for using the waffle flatten that generates the `waffle_flatten` directory